### PR TITLE
downgraded redis version, added support for sslSkipVerify

### PR DESCRIPTION
**Pull request by sredxny**
Fixes https://github.com/TykTechnologies/tyk-identity-broker/issues/102

Was added a new config to enable/disable SSLInsecureSkipVerify for the HTTP server under HttpServerOptions.SSLInsecureSkipVerify

### CODE REVIEW

**Comment by sredxny** [comment]
Was added a new key in the configs `HttpServerOptions.SSLInsecureSkipVerify` however one already exist but used for HTTP client transport layer, should we use the same?

**Comment by buger** [comment]
What about downgraded redis version in title?

**Comment by sredxny** [comment]
@buger yes, currently the logic of the redis backend is for `go-redis/redis/v8` this is messing the things because:

- According to the documentation of go-redis we need to have go-modules enabled to use it, but, currently tib doesn't have go modules (it was covered in a different PR)
- will difficult the process of update tib in dashboard in the long term, as, to use go-redis we must indicate the version in the import path, as: github.com/go-redis/redis/v8 and go detects a type mismatch when dashboard shares the redis connection to embeded tib as `github.com/go-redis/redis/v8` is a different type than `github.com/go-redis/redis`
- go-redis v8 is in beta version. Is safer to just use v6 that is currently being used by dashboard.

File: main.go [hunk 1]
```diff
@@ -82,20 +83,40 @@ func main() {
...
 
-	if config.HttpServerOptions.UseSSL {
-		mainLogger.Info("Broker Listening on SSL:", listenPort)
-		err := http.ListenAndServeTLS(":"+listenPort, config.HttpServerOptions.CertFile, config.HttpServerOptions.KeyFile, p)
-		if err != nil {
-			mainLogger.Fatal("ListenAndServe: ", err)
+	var tibServer net.Listener
+	if config.HttpServerOptions.UseSSL{
+		log.Info("--> Using SSL (https) for TIB")
+		cert, _:= tls.LoadX509KeyPair(config.HttpServerOptions.CertFile, config.HttpServerOptions.KeyFile)
```
Comment (matiasinsaurralde) [inline main.go:94]: Could be useful to cover errors here

Comment (sredxny) [reply main.go]: @matiasinsaurralde done 👍

**Pull request merged**